components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -14,15 +14,16 @@ import * as Utils from 'utils/utils';
 export default class PostProfilePicture extends React.PureComponent {
     static propTypes = {
         compactDisplay: PropTypes.bool.isRequired,
-        enablePostIconOverride: PropTypes.bool.isRequired,
-        hasImageProxy: PropTypes.bool.isRequired,
+        enablePostIconOverride: PropTypes.bool,
+        hasImageProxy: PropTypes.bool,
```

Comment:
The component was already used without providing `enablePostIconOverride` and `hasImageProxy` props. Like:
```javascript
const avatar = (
            <PostProfilePicture
                compactDisplay={false}
                post={selected}
                userId={selected.user_id}
            />
        );
```
So, had to make them optional because TypeScript was complaining about this 🙂